main.go

```diff
@@ -36,14 +36,21 @@ func init() {
 		"Unix domain socket address that the HTTP API should listen on")
 	flag.StringVar(&config.TCPAddr, "tcpAddr", "",
 		"TCP address that the HTTP API should listen on")
-	brokers := flag.String("brokers", defaultKafkaPeers, "Comma separated list of brokers")
-	zookeeper := flag.String("zookeeper", defaultZookeeperPeers, "Comma separated list of ZooKeeper nodes")
+	brokers := *flag.String("brokers", defaultKafkaPeers, "Comma separated list of brokers")
+	zookeeper := *flag.String("zookeeper", defaultZookeeperPeers, "Comma separated list of ZooKeeper nodes followed by optional chroot")
 	flag.StringVar(&pidFile, "pidFile", defaultPIDFile, "Path to the PID file")
 	flag.StringVar(&loggingJSONCfg, "logging", defaultLoggingCfg, "Logging configuration")
 	flag.Parse()
 
-	config.Kafka.SeedPeers = strings.Split(*brokers, ",")
-	config.ZooKeeper.SeedPeers = strings.Split(*zookeeper, ",")
+	config.Kafka.SeedPeers = strings.Split(brokers, ",")
+
+	chrootStartIdx := strings.Index(zookeeper, "/")
+	if chrootStartIdx >= 0 {
+		config.ZooKeeper.SeedPeers = strings.Split(zookeeper[:chrootStartIdx], ",")
+		config.ZooKeeper.Chroot = zookeeper[chrootStartIdx:]
+	} else {
+		config.ZooKeeper.SeedPeers = strings.Split(zookeeper, ",")
+	}
```

Comment:
I didn't catch this before, but `osSigCh` in `main` should be buffered:

> Package signal will not block sending to c: the caller must ensure that c has sufficient buffer space to keep up with the expected signal rate. For a channel used for notification of just one signal value, a buffer of size 1 is sufficient.

http://golang.org/pkg/os/signal/#Notify